### DISCUSSION TITLE
我们为什么要编纂历史？ | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#17] [Comments]
# 我们为什么要编纂历史？ | 夜天之书

早在今年初，适兕就建立了「开源之史」讨论小组，旨在探讨有关技术、人物、思想、文化等开源世界发生过的事件和演化。 一开始，我对这个小组保持观望态度，既期待能够从中了解一些或许自己还不知道的历史，又对历史研究抱有一定的怀疑。一直以来，我都推崇关注现在，解决实际问题，研究历史尤其是尚未有定论也没有“信史”的“开源之史”，到底有多大的作用，我并不确定。但是，近日来发生的一些事情，让我重新考虑“历史”的价值

https://tisonkun.org/2022/04/20/compiling-history/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
非常好的厘定, 
互联网时代的修史和以往必定不同;

以俺私人史书经验来说, 历史 必须演义,
但是, 得基于真实人物的真实行为,
通过这些言行, 来重新构筑一个逻辑自洽的感人故事, 才能流传下去;

开源之史, 值得自比太史公, 首先要创新的:
0: 如何找到一种证据, 可以不随着互联网的发展而消散
1: 应用那种工具, 可以将 开源史书编撰过程中所有变化, 以及参与者的行为, 也都不可篡改的记录下来
2: 即, 我们对资源历史的剖析/重建过程本身, 也应该是 开源历史 的一部分

**Reply by tisonkun** [reply]
> 如何找到一种证据，可以不随着互联网的发展而消散

[Wayback Machine](http://wayback.archive.org/) 可能是这个方向上的一个努力。“信史”本身的验证方式也是多份史料相互印证以及物证的证明。理论上永远存在的证据似乎不太可能。

> 应用那种工具，可以将开源史书编撰过程中所有变化，以及参与者的行为，也都不可篡改的记录下来

这个听起来是某种形式的区块链应用。

> 我们对资源历史的剖析/重建过程本身，也应该是开源历史的一部分

这个是的。
